### DISCUSSION TITLE
@types/gulp-lesshint

### DISCUSSION THREAD

**Discussion by Dulux-Oz** [DefinitelyTyped/DefinitelyTyped#58110] [Request a new @types package] [answered]
Hi All,

Could someone help a (TypeScript) noob with some advice, please, by answering one (or more) of the following questions:

1. Does anyone have the TypeScript def file for gulp-lesshint, and may I please have a copy?
2. Could someone please write the TypeScript def file for gulp-lesshint?
3. Could someone help me to write the TypeScript def file for gulp-lesshint?
4. Could someone point me towards a different Less linter that works with Gulp and which is TypeScript-useable?
5. Could someone point me towards how to use gulp-lesshint "as it is" with TypeScript?

Thanks in advance,

Dulux-Oz

**Comment by martin-badin** [comment, marked as answer]
PR is ready #58115

**Comment by Dulux-Oz** [comment]
Thanks Martin
